src/util/psbt/mod.rs

```diff
@@ -91,6 +91,30 @@ impl PartiallySignedTransaction {
     }
 }
 
+#[cfg(feature = "base64")]
+mod _base64encoding {
+    use super::{PartiallySignedTransaction, Error};
+    use std::fmt::{Display, Formatter, self};
+    use std::str::FromStr;
+
+    impl Display for PartiallySignedTransaction {
+        fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+            use consensus::encode::serialize;
+            use ::base64::display::Base64Display;
```

Comment:
I'd have putten the uses inside the base64 module, especially the consensus::encode one that's reused in both impls, but totally a minor nit.

Reply:
Will do once will understand with Andrew the strategy of merging such PR (with revert option for major API change)